.storybook/preview.js

```diff
@@ -25,37 +27,55 @@ const I18nProvider = (props) => {
   );
 };
 
+function mockFetch(path, init) {
+  if (path === '/api/orgs/1/people/1' && init === undefined) {
+    return Promise.resolve(
+      new Response(
+        JSON.stringify({
+          data: mockPerson(),
+        })
+      )
+    );
+  } else if (path === '/api/orgs/1/people/1/tags' && init === undefined) {
+    return Promise.resolve(new Response('[]'));
+  } else {
+    return Promise.reject({ error: 'unmocked request', path, init });
+  }
+}
+
+class MockApiClient extends FetchApiClient {
+  constructor() {
+    super(mockFetch);
+  }
+}
```

Comment:
I like this! :D

Reply:
Yay! It's definitely more verbose than the configuration format for storybook-addon-mock but on the other hand it requires no knowledge of any addon to understand exactly what it does.